apps/masterbots.ai/app/api/payment/sumarize/route.tsx

```
@@ -1,37 +1,37 @@
+			})
+		}
+	} catch (error) {
+		console.error('Error creating payment intent:', error)
+		return new Response(JSON.stringify({ error: 'Internal Server Error' }), {
+			status: 500,
+			headers: { 'Content-Type': 'application/json' },
+		})
+	}
+}
 
 function summarizePaymentDetails(confirmationToken: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)